Clarify who can start a discussion on a project (#10114)
github/docs: content/discussions/quickstart.md

## Creating a new discussion

> Any logged-in user who can view the repository can create a discussion.

Reviewer: We usually refer to a "logged-in user" as an "authenticated user". This covers people who authenticate by logging in directly, as well as people who authenticate using other methods, like SSO.

```suggestion
Any authenticated user who can view a repository can create a discussion.
```

Author: Thank you!

> Anyone who can view the repository can create a discussion, thus, for public repositories any logged in user start a discussion.

Reviewer 2: How about:
```suggestion
Any logged-in user with access to a repository can create a discussion.
```

Author: @rcowsill, how about this:
`Any logged-in user who can view the repository can create a discussion.`

Author: I made that change - MR should be updated.
